src/components/Display.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+interface DisplayProps {
+    className?: string
+    value?: string
+    defaultValue?: string
+}
+
+const Display: React.FC<DisplayProps> = ({className, value, defaultValue}) => {
+    return (
+        <span className={className}>{value !== undefined && value !== "" ? value : defaultValue}</span>
+    )
+}
```

Comment:
a: placeholder 텍스트를 보여주는 컴포넌트군요. 👍 다음처럼 개선할 수 있을 것 같아요.

```suggestion
const Display: React.FC<DisplayProps> = ({className, value, defaultValue}) => {
    return (
        <span className={className}>{value || defaultValue}</span>
    )
}
```

Reply:
수정했습니다.